ui/desktop/app/templates/scopes/scope/projects/targets/index.hbs

```diff
@@ -68,68 +69,83 @@
             <B.Tr>
               <B.Td>
                 <span class='hds-font-weight-semibold'>
-                  {{#if (can 'read model' B.data)}}
+                  {{#if (can 'read model' B.data.target)}}
                     <LinkTo
-                      data-test-visit-target={{B.data.id}}
+                      data-test-visit-target={{B.data.target.id}}
                       @route='scopes.scope.projects.targets.target'
-                      @model={{B.data.id}}
+                      @model={{B.data.target.id}}
                       @query={{hash isConnecting=false}}
                     >
-                      {{B.data.displayName}}
+                      {{B.data.target.displayName}}
                     </LinkTo>
                   {{else}}
-                    {{B.data.displayName}}
+                    {{B.data.target.displayName}}
                   {{/if}}
-                  {{#if B.data.description}}
-                    <p>{{B.data.description}}</p>
+                  {{#if B.data.target.description}}
+                    <p>{{B.data.target.description}}</p>
                   {{/if}}
                 </span>
               </B.Td>
               <B.Td>
                 <Copyable
-                  @text={{B.data.id}}
+                  @text={{B.data.target.id}}
                   @buttonText={{t 'actions.copy-to-clipboard'}}
                   @acknowledgeText={{t 'states.copied'}}
                 >
-                  <code>{{B.data.id}}</code>
+                  <code>{{B.data.target.id}}</code>
                 </Copyable>
               </B.Td>
               <B.Td>
-                {{#if B.data.type}}
+                {{#if B.data.sessions.length}}
```

Comment:
I do have a question about `B.data.sessions` I didn't saw on the model above we return `sessions` ? how we are getting `B.data.sessions` then?

Reply:
In the model hook we are returning `{ targets: targetsWithSessions, totalItems };` and in the hds table we pass in `@model.targets` for the `@model` arg. `targets` is an array of objects and each object contains a target and the sessions belonging to that target. Does that answer the question or am I misunderstanding?

Reply:
Ahh gotcha that makes total sense to me 😉